<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 88, 182)</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>625</width>
     <height>831</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 11, 192)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="storeButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>70</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius: 24px;
background-color: rgb(59, 171, 232);
color: rgb(63, 63, 63);
}

QPushButton:hover{
		background-color: rgb(128, 103, 255);

}</string>
    </property>
    <property name="text">
     <string>üè™</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>43</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Stores</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>180</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Light</family>
     <pointsize>43</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Health</string>
   </property>
  </widget>
  <widget class="QPushButton" name="healthButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>250</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 24px;
background-color: rgb(59, 171, 232);
color: rgb(63, 63, 63);
}

QPushButton:hover{
	background-color: rgb(128, 103, 255);
}</string>
   </property>
   <property name="text">
    <string>ü©∫</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
